Dockerfile

```diff
@@ -61,7 +61,10 @@ RUN curl -fsSL https://packages.microsoft.com/config/debian/10/packages-microsof
   && curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s --  --profile minimal -y \
   && cargo --version \
   && cargo install cargo-hack \
-  && gem install cocoapods --no-document \
+  && gem install cocoapods -v 1.13.0 \
```

Comment:
this should have a fully pinned gemfile instead of pinning a handful of things -- this is going to be fragile as written